shared/actions/platform-specific/index.desktop.js

```diff
@@ -132,16 +132,19 @@ const checkRPCOwnership = () =>
                 reject(error)
                 return
               }
-              const result = JSON.parse(stdout.toString())
-              if (result.isOwner) {
-                resolve()
+              const resultjs = JSON.parse(stdout.toString())
+              if (resultjs.isOwner) {
+                resolve(resultjs)
                 return
               }
               logger.info(`pipeowner check result: ${stdout.toString()}`)
-              reject(new Error('pipeowner check failed'))
+              throw new Error('pipeowner check failed')
```

Comment:
Would this do it - decrement the handshake wait in the catch{} below?

Generally I struggled trying to invoke the handhsakewait inside the promise, it just didn't seem to work. Open to suggestions.